src/platform/forms-system/src/js/utilities/data/profile.js

```diff
@@ -94,9 +96,15 @@ export const getContent = (appName = 'application') => ({
   state: 'State',
   province: 'Province',
   postal: 'Postal code',
+  zipCode: 'Zip code',
 
   // Error on review & submit
   missingEmailError: 'Missing email address',
+  missingStreetAddress: 'Missing street address',
+  missingCity: 'Missing city',
+  missingStateOrProvince: isUS => `Missing ${isUS ? 'state' : 'province'}`,
+  // international postal code is optional
+  missingZip: isUS => `Missing ${isUS ? 'zip code' : ''}`,
```

Comment:
If `isUS` is false, this error would say `Missing`. Just wanted to doublecheck whether that was an issue.

Reply:
Nice catch! I'll fix it!